<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 234, 199);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>workspace</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>60</y>
        <width>321</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="client">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="code">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>kj</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="update">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
      <widget class="QLabel" name="nomlabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Client</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Emp</string>
       </property>
      </widget>
      <widget class="QComboBox" name="emp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>300</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Salle</string>
       </property>
      </widget>
      <widget class="QComboBox" name="salle">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>340</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="description">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="type">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>sportif</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>culturel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>marriage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>congre</string>
        </property>
       </item>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>220</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="map">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>map</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche">
      <property name="geometry">
       <rect>
        <x>589</x>
        <y>20</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tableau">
      <property name="geometry">
       <rect>
        <x>349</x>
        <y>61</y>
        <width>431</width>
        <height>421</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stat </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Tous</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Code</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Description</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_Client</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_Emp</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Salle</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="calander">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>calendrier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Find Me</string>
     </attribute>
     <widget class="WebAxWidget" name="WebBrowser">
      <property name="control" stdset="0">
       <string>{8856F961-340A-11D0-A96B-00C04FD705A2}</string>
      </property>
      <property name="focusPolicy" stdset="0">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="geometry" stdset="0">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>WebAxWidget</class>
   <extends>QAxWidget</extends>
   <header>webaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
